appcues/src/main/java/com/appcues/debugger/AppcuesDebuggerManager.kt

```diff
@@ -80,7 +80,7 @@ internal class AppcuesDebuggerManager(context: Context, private val koinScope: S
         // control native back press by enabling and disabling on our side
         setDebuggerBackPressCallback(activity)
         // also to make sure that if they change fragment, we will register again after the new fragment is attached
-        (activity as FragmentActivity).supportFragmentManager.addFragmentOnAttachListener { _, _ ->
+        (activity as? FragmentActivity)?.supportFragmentManager?.addFragmentOnAttachListener { _, _ ->
```

Comment:
was there any specific version where this cast didnt work? I wasnt really aware of it. nice

Reply:
In the UI test project, the MainActivity derives from ComponentActivity - I believe since its a Compose-only app - so this cast fails.

Reply:
well possibly the back navigation is not working then.

Reply:
ok, ill merge this to avoid the crash, but look into that.